app/src/main/res/layout-land/category_edit_button.xml

```diff
@@ -1,81 +1,69 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<com.willowtree.vocable.customviews.VocableConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
-    android:id="@+id/parent_layout"
+    android:id="@+id/category_container"
     android:layout_width="0dp"
-    android:layout_height="0dp"
-    android:orientation="vertical"
-    android:theme="@style/SettingsTheme"
-    android:layout_rowWeight="1"
-    android:layout_columnWeight="1">
+    android:layout_height="match_parent"
```

Comment:
So you might still need the `android:layout_rowWeight="1"` and `android:layout_columnWeight="1"` attributes from the original parent view. I'd run that and double check.